### PR TITLE
Docs: Typo in Alibaba Cloud Compute Services Guide

### PR DESCRIPTION
**Pull request by tasoint**
##### SUMMARY
In the Alibaba Cloud Compute Services Guide page, What should be written as vars_files is written as vras_file.

page(https://docs.ansible.com/ansible/devel/scenario_guides/guide_alicloud.html)

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME
Noted statements in the Authentication section.

##### ADDITIONAL INFORMATION
Correct vars_file to vars_files.

### CODE REVIEW

File: docs/docsite/rst/scenario_guides/guide_alicloud.rst [hunk 1]
```diff
@@ -35,13 +35,13 @@ To pass authentication credentials as environment variables::
     export ALICLOUD_ACCESS_KEY='Alicloud123'
     export ALICLOUD_SECRET_KEY='AlicloudSecret123'
 
-To store authentication credentials in a vars_file, encrypt them with :ref:`Ansible Vault<vault>` to keep them secure, then list them::
+To store authentication credentials in a vars_files, encrypt them with :ref:`Ansible Vault<vault>` to keep them secure, then list them::
```
Comment (Akasurde) [inline docs/docsite/rst/scenario_guides/guide_alicloud.rst]:
```suggestion
To store authentication credentials in a vars_files, encrypt them with :ref:`Ansible Vault<vault>` to keep them secure, then list them:

.. code-block:: yaml

```

File: docs/docsite/rst/scenario_guides/guide_alicloud.rst [hunk 2]
```diff
@@ -35,13 +35,13 @@ To pass authentication credentials as environment variables::
...
 
     ---
     alicloud_access_key: "--REMOVED--"
     alicloud_secret_key: "--REMOVED--"
 
-Note that if you store your credentials in a vars_file, you need to refer to them in each Alicloud module. For example::
+Note that if you store your credentials in a vars_files, you need to refer to them in each Alicloud module. For example::
```
Comment (Akasurde) [inline docs/docsite/rst/scenario_guides/guide_alicloud.rst]:
```suggestion
Note that if you store your credentials in a vars_files, you need to refer to them in each Alicloud module. For example:

.. code-block:: yaml
```

**Review by Akasurde — CHANGES_REQUESTED** [review]

**Comment by samccann** [comment]
Thanks @tasoint for the proposed Ansible docs fix! If you select `commit suggestion` on each of the proposed changes above, we can merge this PR.  
image

**Review by Akasurde — APPROVED** [review]

**Review by oraNod — APPROVED** [review]

**Comment by oraNod** [comment]
Thanks for the contribution @tasoint! More contributions are welcome and you can grab one of the [Hacktoberfest issues here](https://github.com/ansible/ansible/labels/hacktoberfest).

**Pull request merged**